app/templates/multi-select.hbs

```diff
@@ -11,14 +17,25 @@
 </div>
 
 <div class="dropdown-menu js-dropdown-menu">
-  {{view listView
-    tagName="ul"
-    classNames="ember-select-results"
-    heightBinding="dropdownHeight"
-    rowHeightBinding="rowHeight"
-    contentBinding="groupedContent"
-    itemViewClassBinding="itemView"
-  }}
+  <ul class="ember-select-results" style={{collectionStyle}}>
+    {{#if showDropdown}}
```

Comment:
This is a good hack, though it makes me wonder if we should have VC add a mutation observer or something for the parent to schedule updates. We've also talked about having a `registerAPI` action that sends up a manual API, could have a `scheduleUpdate` call so you can manually trigger a rerender.

Reply:
This is indeed a hack, I'll add a comment. Basically it is because there is jQuery logic managing the dropdown open/close state.

Ideally you *never* render DOM the user does not see. I think the real fix here is simple: Stop using jQuery to manage the open/closed dropdown state. But I'm not inclined to make that change during this initial effort.

Thus, the hack.

Reply:
You could go ahead and switch it to our new dropdown :trollface: